docs/tools/tikv-control.md

```diff
@@ -253,28 +253,28 @@ success!
 ```
 
 > **Note**:
-> 
+>
 > - This command only supports the local mode. It prints `success!` when successfully run.
 > - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
 > - You need to run this command for all stores where specified Regions' peers are located.
 
 ### Ldb Command
 
-The ldb command line tool offers multiple data access and database administration commands. Some examples are listed below. 
-For more information, refer to the help message displayed when running `tikv-ctl ldb` without any arguments or check the documents from RocksDB.
+The ldb command line tool offers multiple data access and database administration commands. Some examples are listed below.
+For more information, refer to the help message displayed when running `tikv-ctl --ldb help` or check the documents from RocksDB.
 
 Examples of data access sequence:
 
 To dump an existing RocksDB in HEX:
 
 ```bash
-$ tikv-ctl ldb --db=/tmp/test_db dump --hex > /tmp/dbdump
+$ tikv-ctl --ldb='--hex --db=/tmp/db dump'
```

Comment:
em, IMO, I like the origin way - treating all the following args as the ldb args.

Reply:
Yes, I like that way too, but it makes us impossible to parse other arguments besides ldb, which is necessary to support encryption later.

Reply:
How about treating all the args following `--` as the ldb args
like `tikv-ctl ldb xxx -- --hex --db=/tmp/db dump`

Reply:
But I don't think it is better, it implies that all the arguments behind `--` must be the arguments for ldb.

Reply:
seem it can't work if the db path has spaces.

Reply:
why not providing another tool like tikv-ldb?

Reply:
I think this way is with common sense. Such like `cargo`, it uses`cargo test -- --xxx` to pass `--xxx` to test library.

Reply:
Done, I figure out a better solution to allow "ldb" as an external command with `clap`.